OpenRA.Mods.Common/Traits/World/EditorActionManager.cs

```diff
@@ -114,6 +114,11 @@ public void Forward(int id)
 			while (undoStack.Peek().Id != id)
 				Redo();
 		}
+
+		public bool IsModified()
```

Comment:
I think `HasUnsavedItems` would be better naming.

Reply:
Done.